requirements.txt

```diff
@@ -26,3 +26,4 @@ python-dateutil==2.2
 pytz==2014.7
 django-tinsel==0.1.1
 django-url-tools==0.0.8
+django-redis-cache==1.6.0
```

Comment:
We have more experience with ``django-redis`` on other projects. When quickly writing up issue #2256 I referenced the wrong cache interface library. The Django cache framework abstracts this library choice, so switching in ``django-redis`` should be relatively transparent.